selfdrive/ui/qt/onroad/model.cc

```diff
@@ -106,6 +106,12 @@ void ModelRenderer::drawLaneLines(QPainter &painter) {
 }
 
 void ModelRenderer::drawPath(QPainter &painter, const cereal::ModelDataV2::Reader &model, int height) {
+
+  //auto *s = uiState();
+  //auto &sm = *(s->sm);
+  //const auto long_plan_sp = sm["longitudinalPlanSP"].getLongitudinalPlanSP();
+  //bool exp_mode_path = (long_plan_sp.getDynamicExperimentalControl() && long_plan_sp.getMpcSource() == cereal::MpcSource::BLENDED) ||
+  //                     (!long_plan_sp.getDynamicExperimentalControl() && sm["selfdriveState"].getSelfdriveState().getExperimentalMode());
```

Comment:
Why is this commented? is it not needed? should we keep it here if not?

Reply:
@sunnyhaibin is going to add it back :)